lib/server/services/qualityControlFlag/QcFlagService.js

```diff
@@ -318,19 +317,30 @@ class QcFlagService {
                     detectorId: summaryDb.detectorId,
                     effectiveRunCoverage: parseFloat(summaryDb.get('effectiveRunCoverage'), 10) || null,
                     bad: Boolean(summaryDb.get('bad')),
-                    missingVerificationsCount: parseInt(summaryDb.get('missingVerificationsCount'), 10),
+                    flagIds: (summaryDb.get('flagIds')?.split(',') ?? []).map((id) => parseInt(id, 10)),
                     mcReproducible: Boolean(summaryDb.get('mcReproducible')),
                 }));
 
+        const allFlagsIds = new Set(runDetectorSummaryList.flatMap(({ flagIds }) => flagIds));
+        const notVerifiedFlagsIds = new Set((await QcFlagRepository.findAll({
```

Comment:
Can't you use a group by instead of a set?

Reply:
I am not using Set because there are duplicated, but to have faster filtering
```
const missingVerificationsCount = flagIds.filter((id) => notVerifiedFlagsIds.has(id)).length;
```